nav_msgs/Path path
std_msgs/Bool foundCentroid
geometry_msgs/Point centroid
